### ISSUE TITLE
Writing to baseline file seems to fail the first time due to missing directory

### ISSUE THREAD

**Issue by philwolstenholme** [relative-ci/bundle-stats#4832]
I am getting `ENOENT` the first time I try to create a baseline file:

```
npx bundle-stats --baseline generated/bundle-size/webpack-stats.json
✔ Read Webpack stats files
↓ Baseline stats file is missing, see "baseline" option. [SKIPPED]
✖ ENOENT: no such file or directory, open '/Users/wolsph/Projects…
◼ Process data
◼ Generate reports
◼ Save reports
ENOENT: no such file or directory, open '/Users/wolsph/Projects/shop-next/node_modules/.cache/bundle-stats/baseline.json'
```

However, if I create an empty directory first then things are happier:
```
❯ mkdir -p node_modules/.cache/bundle-stats
❯ npx bundle-stats --baseline generated/bundle-size/webpack-stats.json
✔ Read Webpack stats files
↓ Baseline stats file is missing, see "baseline" option. [SKIPPED]
✔ Write baseline data (node_modules/.cache/bundle-stats/baseline.…
✔ Process data
✔ Generate reports
✔ Save reports

┌─────────────────────────────────────┐
│                                     │
│   Bundle Size — 18.39MiB (+100%).   │
│                                     │
└─────────────────────────────────────┘

Artifacts:
- dist/bundle-stats.html
```

**Comment by philwolstenholme** [comment]
There's a chance this directory _was_ actually created at some point but then I did a `rm -rf node_modules/.cache/bundle-stats` while experimenting, unfortunately I can't remember. I'd suggest creating the directory if it's empty though?

**Comment by vio** [comment]
@philwolstenholme thank you for opening the issue!

This is a regression caused by swapping `fs-extra` with `fs`. `fs-extra` handled the missing directory creation by default.

A fix is available for testing on `v4.17.0-beta.7`. Let me know how it goes if you get the chance to try it.

**Comment by vio** [comment]
Released on [`v4.17.0`](https://github.com/relative-ci/bundle-stats/releases/tag/v4.17.0)
